core/src/impl/Kokkos_Error.cpp

```diff
@@ -64,7 +71,8 @@ std::string human_memory_size(size_t arg_bytes) {
 
 void Experimental::RawMemoryAllocationFailure::print_error_message(
     std::ostream &o) const {
-  o << "Allocation of size " << Impl::human_memory_size(m_attempted_size);
+  o << "Allocation of size "
+    << ::Kokkos::Impl::human_memory_size(m_attempted_size);
```

Comment:
I was getting error on my local build.  I do not understand why it was fine before and what changed now but it is not something worth digging into.